### DISCUSSION TITLE
Difference when using a timing metric to count events

### DISCUSSION THREAD

**Discussion by sonoman** [go-kit/kit#1150] [Q&A]
In a project we're using Timing struct. to record latency for a method, with a sample rate of 1.
We send the metrics to Datadog. we also write a log line each time the method executes.
After a while, we graph the number of requests using **duration.count** for the metric, and we find a difference between datadog and the number of logs written for a given minute: approximately 9k min on datadog, and 12k min on the log lines count.

If we use a counter, the metric looks just the same in datadog and logs....but we want to use timing because our primary objective is to measure times. But we also. find a hurdle to add both metrics, given that Timing should do the count just right.

My question is. is this a problem of the library ? or statsd agent ? or maybe Datadog itself ?

**Comment by peterbourgon** [comment]
Does the logger provided to statsd.New log any errors?
